Backend/app/rider-platform/rider-app/Main/src/API/UI/SpecialAssistance.hs

```diff
@@ -0,0 +1,32 @@
+module API.UI.SpecialAssistance
+  ( API,
+    handler,
+    ListRes,
+  )
+where
+
+import qualified Domain.Action.UI.SpecialAssistance as DSpeacialAssistance
+import qualified Domain.Types.Merchant as Merchant
+import qualified Domain.Types.Person as Person
+import qualified Domain.Types.SpecialAssistance as DD
+import Environment
+import Kernel.Prelude
+import Kernel.Types.Id
+import Kernel.Utils.Common
+import Servant
+import Tools.Auth
+
+type API =
+  "disability"
+    :> ( "list"
+           :> TokenAuth
+           :> Get '[JSON] ListRes
+       )
+
+type ListRes = [DD.DisabilityAPIEntity]
```

Comment:
Better naming can be used, infact just use `[DD.DisabilityAPIEntity]` instead of `ListRes`?

Reply:
Done.